server/resources/product/productApi.js

```diff
@@ -5,10 +5,10 @@ const { requireLogin, requireAccountAccess } = require('../../global/handlers/au
 module.exports = (router) => {
 
   router.get('/api/products/default', product.getDefault)
-  router.get('/api/products/:id', requireLogin, product.getSingleById)
```

Comment:
removed for a reason?

Reply:
Just to match the front end configuration.